src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -22,6 +22,7 @@
   height: @ICON_REGULAR_SIZE;
   border: @size_4xs solid @neutral_silver;
   border-radius: @size_2xs;
+  box-sizing: content-box;
```

Comment:
Sneaking in this one which is unrelated to the whitespace issue

Reply:
Nice nice